testing/src/test/java/io/stargate/it/BaseStorageIntegrationTest.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright The Stargate Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.stargate.it;
+
+import com.datastax.oss.driver.api.core.Version;
+import com.datastax.oss.driver.api.testinfra.ccm.CcmBridge;
+
+/**
+ * Base class for tests that need a backend database cluster managed by {@code ccm}.
+ *
+ * <p>Note: the cluster is created on class initialization and is destroyed at JVM exit.
+ */
+public class BaseStorageIntegrationTest {
+
+  public static final String CLUSTER_NAME = "Test_Stargate_Cluster";
+
+  private static final String CCM_VERSION = "ccm.version";
+  private static final boolean EXTERNAL_BACKEND =
+      Boolean.getBoolean("stargate.test.backend.use.external");
+
+  private static final int clusterNodes = Integer.getInteger("stargate.test.backend.nodes", 1);
+  private static final CcmBridge ccm;
+
+  static {
+    String version = System.getProperty(CCM_VERSION, "3.11.8");
+    System.setProperty(CCM_VERSION, version);
+
+    ccm =
+        CcmBridge.builder()
+            .withCassandraConfiguration("cluster_name", CLUSTER_NAME)
+            .withNodes(clusterNodes)
+            .build();
+
+    if (!EXTERNAL_BACKEND) {
+      ccm.create();
+      ccm.start();
```

Comment:
we will start now the ccm for every test case, right now we don't have many ITs but in the future it may impact build time. We may consider creating one shared instance, but also give the possiblity for more advanced setups, see:
https://github.com/stargate/stargate/pull/114#pullrequestreview-494231979

Reply:
Actually, it starts one ccm cluster per test JVM. I'm planning to combine all `test` runs in one JVM in a follow-up PR... did not want to make too many changes at once :)